### PR TITLE
Fastlane

### PR DESCRIPTION
**Pull request by IzzySoft**
You might also wish to move the screenshots from `assets/screenshots/` to `fastlane/metadata/android/en-US/images/phoneScreenshots/`, and the featureGraphic to `fastlane/metadata/android/en-US/images/featureGraphic.png` :wink:

### CODE REVIEW

File: fastlane/metadata/android/en-US/short_description.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+tells you the weather where you are NOT
```
Comment (dkter) [inline fastlane/metadata/android/en-US/short_description.txt]: Would it be okay if I changed this to the short description I have in the GH repo and the Play Store? ("The weather app that tells you the weather where you aren't")
or is there a reason you changed it?

File: fastlane/metadata/android/de/full_description.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<p>Jeden Tag zeigt <i>Elsewhere</i> das Wetter von einem anderen, zufällig gewählten Ort.</p>
+<p><br><b>WARUM?</b></p>
+<p>Na ja, wie das Wetter an Deinem gegenwärtigen Standort ist, siehst Du ja selbst. Was Du jedoch nicht siehst (es sei denn, Du bist genau dort), ist das Wetter in Concordia, Argentinien oder Yingshang Chengguanzhen, China.</p>
+<p>Mit <i>Elsewhere</i> kannst Du Orte entdecken, von denen Du nie zuvor gehört hast. Genau genommen: Streich das; ich kann Dir <i>garantieren,</i>, dass Du das wirst. Eigentlich fast jeden Tag. Die Welt ist riesig – und die Zeit reicht einfach nicht aus, jede einzelne Ecke davon zu besuchen. Vielleicht jedoch reicht sie, um sich weiterer Orte bewusst zu werden.</p>
+<p><br>Die Wetterdaten stammen von <a href='https://openweathermap.org/' target='_blank'>OpenWeatherMap</a>.</p>
```
Comment (dkter) [inline fastlane/metadata/android/de/full_description.txt:5]: Thanks for adding German translations by the way, much appreciated!

File: fastlane/metadata/android/en-US/short_description.txt (hunk 1 above)
Comment (IzzySoft) [reply fastlane/metadata/android/en-US/short_description.txt]: Well, it's all yours – I hear and obey :smile: I don't remember, TBH. And of course you are free to change it anytime. I've just pushed the change.

File: fastlane/metadata/android/de/full_description.txt (hunk 2 above)
Comment (IzzySoft) [reply fastlane/metadata/android/de/full_description.txt:5]: Gladly!

File: fastlane/metadata/android/en-US/short_description.txt (hunk 1 above)
Comment (dkter) [reply fastlane/metadata/android/en-US/short_description.txt]: thank you!

**Comment by IzzySoft** [comment]
PS: should you add more files to fastlane (like mentioned screenshots), please let me know, so I tell my updater to fetch them as well. Thanks!

**Comment by dkter** [comment]
Thanks, I just added the screenshots and feature graphic.

**Comment by IzzySoft** [comment]
Thanks a lot! Updated metadata so they get fetched with your next release.

**Pull request merged**